### DISCUSSION TITLE
Custom Builds - using terser

### DISCUSSION THREAD

**Discussion by btakita** [ai/size-limit#341] [Q&A] [answered]
Hi, I have a [PR](https://github.com/vanjs-org/van/pull/163) to add size-limit to [vanJS](https://github.com/vanjs-org/van). Van has a custom build & uses terser. Is it possible configure size-limit to use build tools such as `terser`?

Right now, I think we can run the custom build then run size-limit on the built js file.

**Comment by ai** [comment, marked as answer]
Yes, just use `@size-limit/file` and run Size Limit on files after your own build script. Something like:

```js
{
  "scripts": {
    "build": "anything",
    "test": "npm build && size-limit"
  },
  "size-limit": [
    {
      "path": "dist/*.js",
    }
  ]
}
```
